service/rtc/msg.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2022-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package rtc
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/pion/webrtc/v3"
+)
+
+type MessageType int
+
+const (
+	ICEMessage MessageType = iota + 1
```

Comment:
just curious, why +1? Is 0 reserved (or implicitly means nil)?

Reply:
0 being the default value for a integer variable I tend to avoid using it so that the event has to be explicitly set.